openedx/core/djangoapps/user_api/views.py

```diff
@@ -153,6 +154,10 @@ def post(self, request):
         from student.views import login_user
         return shim_student_view(login_user, check_logged_in=True)(request)
 
+    @method_decorator(sensitive_post_parameters("password"))
```

Comment:
@stvstnfrd Just saw this now. I used double quotes because that's what's used throughout that particular file. Should I change this to single quotes or leave it as-is? (This is also how it is on upstream now).